src/drivers/PortFolder/WDMTestingTemplate/driver/fail_driver1.c

```diff
@@ -0,0 +1,373 @@
+/*++
+
+Copyright (c) Microsoft Corporation.  All rights reserved.
+
+Module Name:
+
+    fail_driver1.c
+
+Abstract:
+
+    This is a sample driver that contains intentionally placed

```

Comment:
You should update this description to note this sample is expected to be used with your test snippets, and that it contains injected CodeQL defects.

Reply:
fixed.